source/board/6LoWPAN_BorderRouterETHERNET.c

```diff
@@ -19,4 +19,17 @@
  * limitations under the License.
  */
 
-const char *board_id = "7404";
+#include "target_board.h"
+#include "target_family.h"
+
+extern target_cfg_t target_device;
```

Comment:
Because the board can select a particular target_device(s) during runtime.

Reply:
Additional: a null target_cfg means that the target has no flash algo.

Reply:
Sorry, I meant, why was the `target_device` extern removed from the header it was in? It can still be accessed by the board file. Unless there is a really good reason, you should never have externs in a .c/.cpp file because it is a maintenance issue, with lots of duplicated definitions instead of a single one.